### ISSUE TITLE
Empty Wallabag Feed shows a ⚠️ 

### ISSUE THREAD

**Issue by chevdor** [ViennaRSS/vienna-rss#1790]
**Describe the bug**
Using a valid subscription on a Wallabag feed that has no content shows a ⚠️ 

**To Reproduce**
- create a wallabag feed
- read all entries
- refresh in Vienna

**Screenshots**
<img width="281" alt="image" src="https://github.com/user-attachments/assets/4a413dd2-bb11-43d2-a10b-a6c4a2d59f2c">

**Please complete the following information:**
 - Vienna version  3.9.1 :a82691eb: (8219)
 - OS version: Sonoma 14.5 (23F79)

I mentioned Wallabag since I see the issue there but it may or not be related to Wallabag.

<details><summary>Response of an empty feed</summary>

<img width="281" alt="image" src="https://github.com/user-attachments/assets/4a413dd2-bb11-43d2-a10b-a6c4a2d59f2c">

```
<?xml version="1.0" encoding="utf-8"?>
<feed xmlns="http://www.w3.org/2005/Atom">
            <title>wallabag â€” unread feed</title>
        <subtitle type="html">Atom feed for unread entries</subtitle>
        <id>wallabag:wallabag.redacted.org:will:unread</id>
        <link rel="alternate" type="text/html" href="https://wallabag.redacted.org/unread/list"/>
            <link rel="self" type="application/atom+xml" href="https://wallabag.redacted.org/feed/will/<redacted_token>/unread"/>
    <link rel="last" href="https://wallabag.redacted.org/feed/will/<redacted_token>/unread/1"/>
    <generator uri="https://wallabag.org" version="2.6.9">wallabag</generator>
    <author>
        <name>will</name>
    </author>
    <icon>https://wallabag.redacted.org/favicon.ico</icon>
    <logo>https://wallabag.redacted.org/bundles/wallabagcore/themes/_global/img/logo-square.png</logo>
    </feed>
```
</details>

<details><summary>Response of a non empty feed</summary>

<img width="282" alt="image" src="https://github.com/user-attachments/assets/441fd164-97e0-47dc-ac2d-049adbae1af3">

```
<?xml version="1.0" encoding="utf-8"?>
<feed xmlns="http://www.w3.org/2005/Atom">
            <title>wallabag â€” unread feed</title>
        <subtitle type="html">Atom feed for unread entries</subtitle>
        <id>wallabag:wallabag.redacted.org:will:unread</id>
        <link rel="alternate" type="text/html" href="https://wallabag.redacted.org/unread/list"/>
                <updated>2024-08-10T12:03:59+00:00</updated>         <link rel="self" type="application/atom+xml" href="https://wallabag.redacted.org/feed/will/<redacted_token>/unread"/>
    <link rel="last" href="https://wallabag.redacted.org/feed/will/<redacted_token>/unread/1"/>
    <generator uri="https://wallabag.org" version="2.6.9">wallabag</generator>
    <author>
        <name>will</name>
    </author>
    <icon>https://wallabag.redacted.org/favicon.ico</icon>
    <logo>https://wallabag.redacted.org/bundles/wallabagcore/themes/_global/img/logo-square.png</logo>
        <entry>
        <title><![CDATA[GitHub - ViennaRSS/vienna-rss: Vienna is a free and open-source RSS/Atom newsreader for macOS.]]></title>
        <link href="https://github.com/ViennaRSS/vienna-rss"/>
        <link rel="alternate" type="text/html"
              href="https://wallabag.redacted.org/view/329"/>
        <link rel="via"
              href="https://github.com/ViennaRSS/vienna-rss"/>
        <id>wallabag:wallabag.redacted.org:will:entry:329</id>
        <updated>2024-08-10T12:03:59+00:00</updated>
        <published>2024-08-10T12:03:59+00:00</published>
                        <author>
            <name>ViennaRSS</name>
        </author>
                <content type="html" xml:lang="en">
            <![CDATA[estimated reading time: &lt; ...]]>
        </content>
    </entry>
    </feed>
```

</details>

**Comment by barijaona** [comment]
This is deliberate.
Details of feeds are logged in Activity window: that includes reasons for marking them with a ⚠️

**Comment by chevdor** [comment]
In the activity window, I see:
<img width="346" alt="image" src="https://github.com/user-attachments/assets/3ed6721c-343a-4d51-b85b-09e4f6e0c971">

Why would a feed with no content raise a warning ?

**Comment by barijaona** [comment]
As far as I remember, the intent was to warn of a stale feed.
I might change this: keep the log message but suppress the ⚠️

Opinions @ViennaRSS/maintainers ?

**Comment by chevdor** [comment]
This is my first issue, I take the chance to thank you and congratulate for the very nice piece of software 🙏 

I think the log is totally fine as it **may** in some case be an important info to help spot issues.
Some feeds however may be empty and that's totally fine.

As a (new) user, I saw the warning and concluded that something was wrong with my config (url, auth, etc...) whereas there was nothing wrong at all. 

In this case, an option could be to show an ℹ️ icon instead but I would not even do that. The info is "the feed is empty" which is obvious when looking at the empty list.

An option could be also to specifically show a `0` on the feed counts but that is likely not very aesthetic and again not really useful.

The ⚠️ makes sense if the request fails (404, 5xx, etc...) but IMO is more confusing than helpful on empty feeds.

So removing the warning for empty feeds **if the request was successful** is IMO the best way to go.

**Comment by barijaona** [comment]
Closed by Version 3.9.2
